test/integration/policy_info_metric_test.go

```diff
@@ -0,0 +1,185 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net/http"
+	"os/exec"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+	"github.com/open-cluster-management/governance-policy-framework/test/common"
+	policiesv1 "github.com/open-cluster-management/governance-policy-propagator/pkg/apis/policy/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	propagatorMetricsSelector = "component=ocm-policy-propagator"
+	ocmNS                     = "open-cluster-management"
+	metricName                = "policy_governance_info"
+	compliantPolicyYaml       = "../resources/policy_info_metric/compliant.yaml"
+	compliantPolicyName       = "policy-metric-compliant"
+	noncompliantPolicyYaml    = "../resources/policy_info_metric/noncompliant.yaml"
+	noncompliantPolicyName    = "policy-metric-noncompliant"
+)
+
+var routeURL string
```

Comment:
Technically this isn't a URL since a URL would contain the protocol (i.e. `https`). So maybe call this `routeHost`?